multiaccounts/settings_wallet/database_test.go

```diff
@@ -0,0 +1,286 @@
+package walletsettings
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/t/helpers"
+)
+
+func setupTestDB(t *testing.T) (*WalletSettings, func()) {
+	db, stop, err := helpers.SetupTestSQLDB(appdatabase.DbInitializer{}, "wallet-settings-tests-")
+	require.NoError(t, err)
+	d := NewWalletSettings(db)
+	return d, func() { require.NoError(t, stop()) }
+}
+
+func TestTokenPrefrences(t *testing.T) {
```

Comment:
Splitting this into multiple tests should help with debugging in case something fails. What do you think?